### ISSUE TITLE
The details of competing methods

### ISSUE THREAD

**Issue by zhi-xuan-chen** [microsoft/BiomedParse#47]
Dear author,

For your competing methods, I have a few questions.

1. For the results of MedSAM and SAM in Fig. 2.a, when the semantic mask contains multiple instances, do you use a single bounding box encompassing the entire mask, or do you provide a separate bounding box for each instance in the ground truth (GT) mask?

2. Grounding DINO is a model trained on general-domain data. If you use it to extract objects from medical images, is this approach reasonable? If so, how do you select the text prompt, and is the type of prompt you use suitable for Grounding DINO? Additionally, do you generate a bounding box for each individual instance within the mask, or do you produce a single bounding box for the entire mask?

**Comment by theodore-zhao** [comment]
Hi, Thanks for your questions! 

1. In Figure 2a we used a single bounding box encompassing the entire mask for the models to output semantic mask. 
2. We understood that Grounding DINO was a generic text prompt detection model and was not specialized for medical images. As the training code for Grounding DINO was not available during the experiments, we used the model as is. The performance difference shows the gap in medical domain specific training. We used the same prompts as for BiomedParse during inference. The box with highest confidence is selected for prompting SAM or MedSAM to produce the entire mask. 

Hope these clarifies your questions!

**Comment by zhi-xuan-chen** [comment]
Thank you for your quick response!

I also want to ask the number of masks you mentioned in your paper is the semantic mask or the instance mask. If one mask contains many instances, will it be counted as 1 or many?

**Comment by theodore-zhao** [comment]
It will be counted as one semantic mask.

**Comment by zhi-xuan-chen** [comment]
Thank you very much!

I noticed that you computed certain metrics for the "object." I just wanted to ask for clarification regarding the definition of the term "object" in this context. Does it refer to a semantic mask for a single category, or does it represent an instance mask?

I would greatly appreciate your guidance on this to fully understand the methodology. Thank you so much!

**Comment by theodore-zhao** [comment]
The "object type" refers to the semantic mask for a single category.

**Comment by zhi-xuan-chen** [comment]
OK, thank you very much!
